<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SMDock</class>
 <widget class="QDockWidget" name="SMDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>380</height>
   </rect>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <property name="windowTitle">
   <string>SMARTGREEN</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tab">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::ElideNone</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="ViewTab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>&amp;View</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QToolBox" name="LayerOptions">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="nodePage">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>652</width>
             <height>215</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Node</string>
           </attribute>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="styleLbl">
              <property name="text">
               <string>Style:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="nodeStyleCB"/>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="showNodeID">
              <property name="text">
               <string>Show ID</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="linkPage">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>156</width>
             <height>61</height>
            </rect>
           </property>
           <attribute name="label">
            <string>Link</string>
           </attribute>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="1">
             <widget class="QComboBox" name="linkStyleCB"/>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="styleLbl_2">
              <property name="text">
               <string>Style:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QCheckBox" name="showLinkID">
              <property name="text">
               <string>Show ID</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="lidPage">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>141</width>
             <height>64</height>
            </rect>
           </property>
           <attribute name="label">
            <string>LID</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <widget class="QCheckBox" name="showLIDID">
              <property name="text">
               <string>Show ID</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="showLIDConnection">
              <property name="text">
               <string>Show LID connection</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="LogTab">
       <attribute name="title">
        <string>Report</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTextBrowser" name="_console"/>
        </item>
        <item>
         <widget class="QProgressBar" name="_progressbar">
          <property name="value">
           <number>24</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Abort|QDialogButtonBox::Reset</set>
          </property>
          <property name="centerButtons">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
